draft-ietf-rtcweb-jsep.xml

```diff
@@ -1243,6 +1243,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           and/or pending remote description according to the rules defined for
           Trickle ICE. Connectivity checks will be sent to the new candidate.</t>
 
+          <t>This method can also be used to provide an end-of-candidates
+          indication to the ICE Agent, as defined in
+          <xref target="I-D.ietf-ice-trickle"/>.</t>
```

Comment:
Should we be more specific about it being the null value that mean end-of-candidates?